_posts/2019-10-??-wprowadzenie-do-zed-attack-proxy.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: Wprowadzenie do Zed Attack Proxy
+published: false
+date:      2019-09-30 03:00:00 +0100
+author:    amak
+tags:
+  - programming
+  - pentest
+  - pentesting
+  - tool
+  - OWASP
+  - security
+---
+
+ZAP (Zed Attack Proxy) jest opensourcowym narzędziem tworzonym przez organizację OWASP wspomagającym testy penetracyjne, które służy do znajdowania podatności bezpieczeństwa w aplikacjach
+webowych. Działa na zasadzie proxy, dzięki czemu pozwala nie tylko na podglądanie żądań wysyłanych do serwera aplikacji i odpowiedzi z serwera otrzymywanych, ale daje również
+możliwość debugowania, modyfikowania oraz wysyłania własnych żądań. Jest narzędziem dostosowanym do obsługi HTTP, oferuje prosty sposób na rozszyfrowanie HTTPS poprzez dodanie własnego
+certyfikatu do przeglądarki. Dostarcza automatyczne skanery, jak również narzędzia pomagające manualnie testować aplikację.
+
+## Instalacja oraz konfiguracja
+
+Link do pobrania:
+
+https://github.com/zaproxy/zaproxy/wiki/Downloads
+
+By móc korzystać ze wszystkich funkcjonalności jakie dostarcza nam ZAP powinniśmy zacząć pracę od konfiguracji przeglądarki, tak by łączyła się ona przez proxy (lokalnie odpalonego ZAPa).
+Z uwagi na prostą konfigurację najczęściej polecaną przeglądarką jest Firefox. W menu przeglądarki odnajdujemy Preferencje -> Sieć i wchodzimy w Ustawienia. Zaznaczamy "Ręczna konfiguracja
```

Comment:
Wyrzuciłbym to "najczęściej"

Reply:
i użyjmy "→" zamiast "->" - wygląda lepiej. Gdzieś dalej nawet już użyłaś strzałeczki ;)